index.html

```diff
@@ -0,0 +1,131 @@
+            <li><a class="navItem" href="#packages">Packages</a></li>
+            <li><a class="navItem" href="./about.html">Join</a></li>
+            <li><a class="navItem" href="#sponsors">Sponsor</a></li>
+            <li><a class="navItem" href="./about.html">Recipes</a></li>
+            <li><a class="navItem navItem-main" href="./about.html">Workshops</a></li>
+          </ul> 
+      </nav>   
+      <nav>
+        <div onclick="menuPopup()" id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div> 
+        <div onclick="menuPopup()" id="closeButton"> &times; </div>
+      </nav>
+    </header>
+
+    <section class="mobilemain">
+        <article class="home-page" id="#home" title="home-page">
+          <h3 class="intro-title-one"><q>cakey-cakey by Marianna</q></h3>
+          <h1 class="intro-title-two">CAKE-LOVERS WORKSHOP FOR MINORITY GROUP 2023</h1>
+            <p class="explanation-paragraph" alt="explanation-paragraph">
+              A workshop that will allows low income individuals to
+              exhibit with low income to get source of income why can sell their cake to the host community and make money out of it</p>
+          <h4 class="event-date" alt="event-date"> 2023.05.25(THU) ~ 26(FRI)</h4>
+          <div class="copyright">&copy; Nairobi, Kenya </div>
+        </article>
+
+        <article id="packages-offered" title="packages-offered">
+            <h5 class="packages-title" id="packages"> Packages Offered</h5>
+
+            <section class="card-stack">
+              <div class="cards card1" alt="Card1">
+                <img class="cards-img card1-img" src="images/program_icon_01.png" alt="Card1-image">
+                  <h5 class="card-title">lecture</h5>
+                  <p class="card-text">Lorem ipsum dolor sit amet conseinus cabo, perferendis veniam neque!.</p>
+              </div>
+
+              <div class="cards card2" alt="Card2">
+                <img class="cards-img card2-img" src="images/program_icon_02.png" alt="Card2-image">
+                  <h5 class="card-title">lecture</h5>
+                  <p class="card-text">Lorem ipsum dolor sit amet conseinus cabo, perferendis veniam neque!</p>
+              </div>
+
+              <div class="cards card3" alt="Card3">
+                <img class="cards-img card3-img" src="images/program_icon_03.png" alt="Card3-image">
+                  <h5 class="card-title">lecture</h5>
+                  <p class="card-text">Lorem ipsum dolor sit amet conseinus cabo, perferendis veniam neque!.</p>
+              </div>
+
+              <div class="cards card4" alt="Card4;">
+                <img class="cards-img card4-img" src="images/program_icon_04.png" alt="Card4-image">
+                  <h5 class="card-title">lecture</h5>
+                  <p class="card-text">Lorem ipsum dolor sit amet conseinus cabo, perferendis veniam neque!ard</p>
+              </div>
```

Comment:
- [OPTIONAL] You did great personalizing the project content. :+1: I can see you wrote about a cake workshop. I think it will also be nice and make your project look more professional and presentable for your portfolio if you add real texts related to the event  here instead of using `Lorem ipsum` placeholder text. :+1: :smile: :wave: